scripts/SortVideos.java

```diff
@@ -0,0 +1,69 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//JAVA 17
```

Comment:
I think 17+ should be ok like for the other script
```suggestion
//JAVA 17+
```